main.tex

```diff
@@ -708,21 +752,21 @@
 
 \item  stored on Common Property except in areas designated by the Owners Corporation or otherwise with its written consent; and
 
-\item  brought into a Lot (other than a Lot designated for parking) or the foyer, lifts, stairwells, hallways, garden areas, walkways, balconies or other Common Property.
+\item  brought into the stairwells or garden areas.
 
 \end{enumerate}
 
-\item  An Owner and Occupiers indemnify and keep indemnified the Owners Corporation on a full indemnity basis for the costs of repairing any damage to the Common Property arising out of bringing their bicycles on to any Lot (for example, corridors).
+\item  Owner and Occupiers indemnify and keep indemnified the Owners Corporation on a full indemnity basis for the costs of repairing any damage to the Common Property arising out of bringing their bicycles on to any Lot (for example, corridors).
 
 \end{enumerate}
 
 \item  Damages to and Cleanliness of Common Property
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must not damage or alter the Common Property without the written approval of the Owners Corporation.
```

Comment:
We have grates everywhere, intended to support the growth of plants, yet altering these grates by growing such plants is against the rules.